deploy-manage/monitor/cloud-health-perf.md

```diff
@@ -0,0 +1,124 @@
+
+## Cluster performance [ec-es-cluster-performance]
+
+```{applies_to}
+deployment:
+  ess:
+```
+
+{{ess}} deployments offer an additional **Performance** page to get further information about your cluster performance. 
+
+If you observe issues on search and ingest operations in terms of increased latency or throughput for queries, these might not be directly reported on the **Health** page, unless they are related to shard health or master node availability. 
+
+The **Performance** page and the out-of-the-box logs allow you to monitor your cluster performance, but for production applications we strongly recommend setting up a dedicated monitoring cluster. Refer to [Understanding deployment health](#ec-health-best-practices), for more guidelines on how to monitor you cluster performance.
+
+[Learn more about the performance page and the metrics it captures](/deploy-manage/monitor/access-performance-metrics-on-elastic-cloud.md).
+
+## JVM memory pressure indicator
+
+{{ech}} and {{ece}} also provide a [JVM memory pressure indicator](/deploy-manage/monitor/ec-memory-pressure.md) for each node in your cluster in your deployment overview. This indicator can help you to determine when you need to upgrade to a larger cluster.
+
+## Preconfigured logs and metrics [ec-es-health-preconfigured]
+
+In a non-production environment, you may choose to rely on the logs and metrics that are available for your deployment by default. The deployment **Logs and metrics** page displays any current deployment health warnings, and from there you can also view standard log files from the last 24 hours.
+
+The logs capture any activity related to your deployments, their component resources, snapshotting behavior, and more. You can use the search bar to filter the logs by, for example, a specific instance (`instance-0000000005`), a configuration file (`roles.yml`), an operation type (`snapshot`, `autoscaling`), or a component (`kibana`, `ent-search`).
+
+:::{tip} Monitoring in production environments
+In a production environment, it’s important set up dedicated health monitoring using [stack monitoring](/deploy-manage/monitor/stack-monitoring/ece-ech-stack-monitoring.md). Stack monitoring allows you to retain the logs and metrics that can be used to troubleshoot any health issues in your deployments. In the event of that you need to [contact our support team](/troubleshoot/index.md#contact-us), they can use the retained data to help diagnose any problems that you may encounter.
+
+You have the option of sending logs and metrics to a separate, specialized monitoring deployment, which ensures that they’re available in the event of a deployment outage. The monitoring deployment also gives you access to {{kib}} stack monitoring features, through which you can view health and performance data for all of your deployment resources.
+:::
```

Comment:
I believe this sub-section only belongs to ECH, as in ECE the preconfigured logs and metrics offered in a different way, as part of [ECE platform monitoring](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/750/deploy-manage/monitor/orchestrators/ece-platform-monitoring).

I think we have to find a way to link both properly.

Anyway this doc (cloud-health-perf) looks awesome, and 90% of the content is valid for both ECE and ECH. Great idea!

Reply:
Also to provide more background, the ECE platform monitoring, which has logs + metrics + proxy data from all deployments, is intended to use by ECE administrators, in contrast with the stack monitoring (logs and metrics) configured at deployment(s) level, which is intented to be useful for the users of the deployments.

If a user belongs only to a deployment and needs to take care of the cluster, they should rely on stack monitoring (logs and metrics) configured at deployment level, as probably they don't have access to the system `logging-and-metrics` cluster that stores the logs and metrics collected at platform level.

Reply:
good catch.